core/src/lib/CMakeLists.txt

```
@@ -154,17 +153,13 @@ set_target_properties(version-obj PROPERTIES POSITION_INDEPENDENT_CODE ON)
 
 target_link_libraries(
   bareos
-  bareosfastlz
-  ${OPENSSL_LIBRARIES}
-  ${PTHREAD_LIBRARIES}
-  ${ZLIB_LIBRARIES}
-  ${ACL_LIBRARIES}
-  ${LZO2_LIBRARIES}
-  ${CAP_LIBRARIES}
-  ${WRAP_LIBRARIES}
-  ${CAM_LIBRARIES}
-  ${WINDOWS_LIBRARIES}
-  ${JANSSON_LIBRARIES}
+  PRIVATE bareosfastlz
+  PRIVATE ${OPENSSL_LIBRARIES} ${PTHREAD_LIBRARIES}
+  PRIVATE ${ZLIB_LIBRARIES}
+  PRIVATE ${ACL_LIBRARIES}
+  PRIVATE ${LZO2_LIBRARIES}
+  PRIVATE ${CAP_LIBRARIES}
+  PRIVATE ${JANSSON_LIBRARIES} ${CAM_LIBRARIES} ${WINDOWS_LIBRARIES}
```

No need to repeat the PRIVATE keyword over and over.
```suggestion
  PRIVATE bareosfastlz
          ${OPENSSL_LIBRARIES} ${PTHREAD_LIBRARIES}
          ${ZLIB_LIBRARIES}
          ${ACL_LIBRARIES}
          ${LZO2_LIBRARIES}
          ${CAP_LIBRARIES}
          ${JANSSON_LIBRARIES}
          ${CAM_LIBRARIES}
          ${WINDOWS_LIBRARIES}
```